examples/standard/cluster-role.yaml

```diff
@@ -126,3 +126,10 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
+  - jobset.x-k8s.io
```

Comment:
That looks like an external CRD that is not part of a default kubernetes install, is it?

Reply:
Correct, this is still a CRD at the moment, but the demand for having jobset metrics is increasing and people are working on making it a native k8s api, so I'm adding it as a build-in resource here.